src/main/java/com/tabnine/TabNineCompletionContributor.java

```diff
@@ -210,7 +210,6 @@ public void fillCompletionVariants(CompletionParameters parameters, CompletionRe
                 result.addLookupAdvertisement(details);
             }
             if (originalMatcher.getPrefix().length() == 0 && completions.results.length == 0) {
-                result.stopHere();
                 return;
```

Comment:
option 2 - explicitly call result.runRemainingContributors(parameters, true); and then result.stopHere();